packages/landingpage-new/components/home/header/install-chips.module.scss

```diff
@@ -1,15 +1,18 @@
 .container {
   display: flex;
-  column-gap: 20px;
+  row-gap: 20px;
+  flex-direction: column;
   position: relative;
 }
 
 .chipContainer {
-  width: 410px;
```

Comment:
![image](https://user-images.githubusercontent.com/22963121/207588183-4b932940-93a6-4933-9140-869264ce434b.png)

I think the chip has to be slightly wider. Also, the copy message should be moved to the right so it doesn't overlap with the chip. Could you also animate the confirmation message? For example, we could make it slide from behind the button. Should be easy to do it like this:
```scss
.successMessage {
  position: absolute;
  color: #68ce6a;
  font-weight: bolder;
  top: -5px;
  left: 0;
  transition: left 1s;

  &Show {
    left: 400px;
  }
}
```

Also necessary:
- increase z-index of chip container
- give chip-container a white background color
- give chip-container witdh `fit-content`

The only thing that's tricky is that the `left` position of the message needs to account for the width of the React chip.

Reply:
i removed the absolute position of the success message and placed it next to the chip. is the animation to the right important, or is it ok with visibility?

Reply:
its fine like this 👍🏼